int32[] motor_index  # 电机编号
float64[] q         # 实际位置
float64[] dq        # 实际速度  
float64[] ddq       # 实际加速度
float64[] tau_est   # 力矩估计